packages/router/src/history.js

```diff
@@ -5,19 +5,19 @@ const createHistory = () => {
     listen: (listener) => {
       const listenerId = 'RW_HISTORY_LISTENER_ID_' + Date.now()
       listeners[listenerId] = listener
-      window.addEventListener('popstate', listener)
+      window && window.addEventListener('popstate', listener)
```

Comment:
Nice new syntax! https://www.joshwcomeau.com/operator-lookup?match=optional-chaining

```suggestion
      window?.addEventListener('popstate', listener)
```

Reply:
I originally did these, because I was still getting window not defined errors (but was due to something else). I also prefer the  conditional syntax!